Source/IGListGridCollectionViewLayout.h

```diff
@@ -10,6 +10,16 @@
 #import <UIKit/UIKit.h>
 #import <IGListKit/IGListMacros.h>
 
+/**
+ An option for how to align items in grid layout.
+ */
+typedef NS_ENUM(NSInteger, IGListGridCollectionViewLayoutAlignment) {
+    IGListGridCollectionViewLayoutAlignDefault,
```

Comment:
- what does `Default` mean? i would expect to see `Left`, `Right`, `Center`... 
- nit: can we spell-out `Alignment` for each enum case? (follows conventions)